api/src/EventListener/RequestTransactionListener.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace App\EventListener;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\Event\ExceptionEvent;
+use Symfony\Component\HttpKernel\KernelEvents;
+
+/**
+ * Wraps a database transaction around all SQL statements in the whole request.
+ */
+final class RequestTransactionListener implements EventSubscriberInterface {
+    public function __construct(private EntityManagerInterface $entityManager) {
+    }
+
+    public static function getSubscribedEvents(): array {
+        return [
+            KernelEvents::CONTROLLER => ['startTransaction', 10],
```

Comment:
Mit disableReboot wird das Controller Event nicht ein zweites mal ausgelöst in den Tests.
Mit dem Controller Event KernelEvents::REQUEST kommt man schon einen schritt weiter, aber es läuft noch nicht ganz